### ISSUE TITLE
No gen_bigquery_reader_ops

### ISSUE THREAD

**Issue by yarons** [tensorflow/tensorflow#26363]
**System information**
- OS Platform and Distribution: Linux Ubuntu 16.04
- TensorFlow installed from: binary
- TensorFlow version: 1.12.0
- Python version: 3.5
- Installed using: pip
- CUDA/cuDNN version: 9.0
- GPU model and memory: Jetson TX2

I can't run uff conversion.
The traceback is:

```
Traceback (most recent call last):
  File "/tegra/AIC_tf_to_trt_image_classification/scripts/frozen_graphs_to_uffs.py", line 9, in <module>
    from model_meta import NETS, FROZEN_GRAPHS_DIR, CHECKPOINT_DIR, UFF_DIR
  File "/tegra/AIC_tf_to_trt_image_classification/scripts/model_meta.py", line 10, in <module>
    import tensorflow.contrib.slim as slim
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/contrib/__init__.py", line 38, in <module>
    from tensorflow.contrib import cloud
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/contrib/cloud/__init__.py", line 24, in <module>
    from tensorflow.contrib.cloud.python.ops.bigquery_reader_ops import *
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/contrib/cloud/python/ops/bigquery_reader_ops.py", line 21, in <module>
    from tensorflow.contrib.cloud.python.ops import gen_bigquery_reader_ops
ImportError: cannot import name 'gen_bigquery_reader_ops'
```
There's no such file `gen_bigquery_reader_ops.py` at this location.

**Comment by jvishnuvardhan** [comment]
@yarons Could you provide more details on the problem and its context? If possible, please provide a code to reproduce the issue that makes it faster to find rootcause of the issue. Thanks!

**Comment by yarons** [comment]
@jvishnuvardhan
https://github.com/tensorflow/tensorflow/blob/afab5b322f780c235c61038b49593e34b523d400/tensorflow/contrib/cloud/python/ops/bigquery_reader_ops.py#L21

This line is referencing this library but I couldn't find it anywhere.

**Comment by gadagashwini** [comment]
@yarons This is fixed in Tensorflow 1.14.0. Would you like to upgrade your Tensorflow version and try. Thanks!

**Comment by yarons** [comment]
I will try that again, thank you :)
